rendercv/cli/commands.py

```diff
@@ -136,6 +144,30 @@ def cli_command_render(
     extra_data_model_override_argumets: typer.Context = None,  # type: ignore
 ):
     """Render a CV from a YAML input file."""
+    if watch:
+        cwd = os.getcwd()
+        def rerun_command():
+            os.chdir(cwd) # Undo the chdir present later in the file for new runs.
+            cli_command_render(
+                input_file_name=input_file_name,
+                use_local_latex_command=use_local_latex_command,
+                output_folder_name=output_folder_name,
+                latex_path=latex_path,
+                pdf_path=pdf_path,
+                markdown_path=markdown_path,
+                html_path=html_path,
+                png_path=png_path,
+                dont_generate_markdown=dont_generate_markdown,
+                dont_generate_html=dont_generate_html,
+                dont_generate_png=dont_generate_png,
+                watch=False,
+                extra_data_model_override_argumets=extra_data_model_override_argumets,
+            )
+
+        input_file_path: pathlib.Path = pathlib.Path(input_file_name).absolute()
+        watcher.run_a_function_if_a_file_changes(input_file_path, rerun_command)
+        return
```

Comment:
We should move this block after 205 so that users can use the `watch` from the YAML file, not necessarily from the CLI. It should look for `render_command_settings_dict["watch"]`, not `watch` directly.